README.adoc

```diff
@@ -140,16 +140,15 @@ To link this JSON file to your application, create a `start/src/main/java/io/ope
 include::finish/src/main/java/io/openliberty/guides/config/CustomConfigSource.java[tags=customConfig]
 ----
 
-The `setProperties()` private method reads the key value pairs from the `CustomConfigSource.json` JSON file and writes the information into a map.
+The `getProperties()` method reads the key value pairs from the `resources/CustomConfigSource.json` JSON file and writes the information into a map.
 
-To register the custom configuration source, create a `start/src/main/resources/META-INF/services/org.eclipse.microprofile.config.spi.ConfigSource` file and add the full class name into it:
+Lastly, it is an important step to register the custom configuration source, create a file called `org.eclipse.microprofile.config.spi.ConfigSource` under `start/src/main/resources/META-INF/services/` directory and add the full class name into it:
```

Comment:
We can simply say "Lastly, register the custom configuration source.  Create a file, `...spi.ConfigSource` under the `start/...` directory and add the full class name into it ...".

Or, "Last but important ...".

Reply:
I don't get why we need "and add the full class name into it:" if user is creating the whole file.